spawn_worker/Cargo.toml

```diff
@@ -6,7 +6,7 @@ version = "0.0.1"
 [dependencies]
 anyhow = { version = "1.0" }
 memfd = { version = "0.6" }
-nix = { version = "0.26" }
+nix = { version = "0.24" }
```

Comment:
Minor: This revert from version 0.26 to 0.24 seems related to CentOS 7. Are there issues using 0.26 on it?

Reply:
Yes there are, its a weird one too. 
Since it links into libc calls/syscalls not available on Centos 7 by default (memfd_create wasn't available in the original kernel shipped with RHEL 7, and in turn its libc) . 

Curiously I wasn't using `memfd_create` from nix, instead I used a different crate, which calls the syscall directly and doesn't link to libc (via rustix). Just including `nix 0.26` was enough to trigger this problem

I need to investigate `memfd` further, to make sure most Centos7 installs in the wild will actually have the updated kernel.

AFAIR native profiler also uses the memfd /cc @r1viollet @sanchda - wonder how this looks on your side

Reply:
Anyway - I've added the Centos7 build to CI, to safeguard against mishaps like this.

Reply:
Well,
```
static inline int memfd_create(const char *name, unsigned int flags) {
  return syscall(SYS_memfd_create, name, flags);
}
```
:)

the `memfd_create()` syscall was added in kernel 3.17, which is basically as far back as the native profiler supports anyway, so this hasn't really been a problem in the profiler.